tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -159,55 +170,67 @@ def test_failed_getenv_vetext_api_endpoint_domain_property(mocker, missing_domai
     assert response['statusCode'] == 200
     sqs_mock.assert_called_once()
 
+
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
```

Comment:
Not part of his PR, but the parentheses around the list elements are unnecessary.  Ditto elsewhere in which the list element is a single value.